nekoyume/Assets/_Scripts/UI/Widget/Popup/ItemCountableAndPricePopup.cs

```diff
@@ -291,10 +287,14 @@ public void Show(Model.ItemCountableAndPricePopup data, bool isSell)
                 SubmitWidget = () => reregisterButton.OnSubmitSubject.OnNext(default);
             }
 
-            countInputField.enabled = isSell;
-            addCountButton.gameObject.SetActive(isSell);
-            addMaximumCountButton.gameObject.SetActive(isSell);
-            removeCountButton.gameObject.SetActive(isSell);
+            var isMulti = data.Item.Value.MaxCount.Value > 1;
```

Comment:
```suggestion
            var isCountableItem = data.Item.Value.MaxCount.Value > 1;
```
I think a more intuitive name would be better.